server/game/Card.js

```diff
@@ -606,9 +607,9 @@ class Card extends EffectSource {
 
         _.each(this.getPersistentEffects(true), (effect) => {
             if (effect.location !== 'any') {
-                if (to === 'play area' && from !== 'play area') {
```

Comment:
This change is in a neurological part of the code

Reply:
Understood, but I needed a way to trigger effect recalculation on a move from discard to hand, where the play area is not involved at all.  Happy to do it a different way if you have any recommendations.